### DISCUSSION TITLE
OnionOS do not pass loading

### DISCUSSION THREAD

**Discussion by Pindakk** [OnionUI/Onion#779] [Q&A] [answered]
Hello guys,

I am having a strange problem. I am following this video https://www.youtube.com/watch?v=IQQi8A6M_zw as a guide.
I have a 32gb kingston SD card, Format fat32, I upgrade the firmware or at least I think so because un the video he get an upgrade image but I did not. after that I put the OnionOS in the card and the miyoo instaled it but after the 100% switch off and got in a constant loop with the Initial screen and a "noice" and back. Any idea?
I was able to factory resest with the old SD.
Many thanks

**Comment by Aemiii91** [comment, marked as answer]
If you didn't see the upgrade screen, then the upgrade didn't work. Try again, preferably with a different card.

**Reply by Pindakk** [reply]
Right, Kingston 32gb SD card did NOT work, and I am triying with a SanDisk 128 and It is working. thanks!
